README.md

```diff
@@ -51,7 +51,7 @@ Stop gatsby and restart with your new region environment by running: `REGION=NEW
 * [x] Find content images
 * [x] Learn how to use a data file to pull data for different components
 * [x] Image optimization
-* [ ] Harm page content
+* [x] Harm page content
```

Comment:
Small typos like that are ok to put into small code reviews like this one, especially since the major piece of work here is essentially "clean up" to other code that wasn't quite correct!

Generally for myself if I have a PR that is really thematically well-defined (like adding in a specific feature) I try not to add typo fixes and stuff into that PR and will later do a "Typos/cleanup" type PR after that. But, the other thing is that we're still people on a team together and for small projects with few contributors, it's not too harmful to sneak in a fix or small readme change like this so it's fine to let it slide. It's always a bit of a trade off in terms of being super strict to the rules and having a perfect git history and also not wanting to make the process so cumbersome and boring that coding isn't fun anymore, you know? Some people are more loosy-goosy with the best practices and some are very strict so you'll have to feel it out whenever you join a new team.

Reply:
Oh yes, makes sense to me! Thanks for the feedback :-)